airflow/www/views.py

```diff
@@ -798,11 +798,14 @@ def log(self):
         if PY2 and not isinstance(log, unicode):
             log = log.decode('utf-8')
 
+        if request.is_xhr:
```

Comment:
I don't like relying on `is_xhr` to do this... it basically uses a non-standard header `X-Requested-With` (http://flask.pocoo.org/docs/0.11/api/#flask.Request.is_xhr) and will for eg break mysteriously if we move away from `$.get`

I think alternatives (from best to worst IMO) are

- Using a separate (maybe behind our API) endpoint
- Use a new query param
- Set `Accept: text/plain` header